.github/workflows/main.yaml

```diff
@@ -349,3 +349,44 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  
+  container_runtime_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        container-runtime: ['containerd', 'cri-o', 'docker']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install and configure container runtime
+        run: |
+          if [ "${{ matrix.container-runtime }}" == "docker" ]; then
+            sudo apt-get update
+            sudo apt-get install \
+                apt-transport-https \
+                ca-certificates \
+                curl \
+                gnupg \
+                lsb-release
+            curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+            echo \
+              "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+              $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+            sudo apt-get update
+            sudo apt-get install docker-ce docker-ce-cli containerd.io
```

Comment:
We need also to install cri-dockerd, KubeEdge remove the support of dockershim since v1.14.

Reply:
✅